packages/contracts-bedrock/contracts/governance/MintManager.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Shouldn't this be a rename?

Reply:
I also cleaned up some stuff in the contract, namely error strings